### PR TITLE
Proposal/Adding Prow Testing

### PR DESCRIPTION
**Pull request by jonahjon**

### CODE REVIEW

File: proposals/20200912-prow-on-aws.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+- Provide distributed method for handling PR merging.
+- Utilize Github PR commands to control testing workflow such as re-tests
+
+
+### Prerequisites
+Install Prow on AWS EKS, the only prerequisite is that we have an AWS account, and an S3 bucket to store artifacts for Prow.
+
+### Design Detail
+
+Propsed Install components
```
Comment (leodido) [inline proposals/20200912-prow-on-aws.md:34]:
```suggestion
Proposed install components
```

File: proposals/20200912-prow-on-aws.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+- Plank
+- Hook
+- GhProxy
+- PodUtilities
+- Horologium
+- Tide (will look into difficulty)
+- Spyglass
```
Comment (leodido) [inline proposals/20200912-prow-on-aws.md:41]: 💘

File: proposals/20200912-prow-on-aws.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+- Crier
+- Testgrid
```
Comment (leodido) [inline proposals/20200912-prow-on-aws.md:43]: 😮  🥇

**Review by leodido — COMMENTED** [review]
Thanks for opening this Jonah!

**Comment by jonahjon** [comment]
> Thanks for opening this Jonah!
> 
> Left a comment about a typo.
> Would you please sign-off your commit? :)

I think I signed it off!

**Pull request closed without merging**